en/advanced_features/traffic_avoidance_utm.md

```diff
@@ -0,0 +1,34 @@
+# Air Traffic Avoidance: MAVLink
```

Comment:
This is probably OK, but we might want to be more specific after we've found out more about this. Ie. Air Traffic Avoidance: MAVLink UTM Global Postion

Reply:
I updated to just be UTM.